assignments/lambda-classes.js

```diff
@@ -1 +1,95 @@
 // CODE here for your Lambda Classes
+class Person{
+    constructor(name, age, location){
+        this.name = name;
+        this.age = age;
+        this.location = location;
+    }
+    speak(){
+        console.log(`Hello my name is ${this.name}, I am from ${this.location}.`);
+    }
+}
+
+class Instructor extends Person{
+    constructor(instructorAttrs){
+        super(instructorAttrs)
+        this.specialty = instructorAttrs.specialty;
+        this.favLanguage = instructorAttrs.favLanguage;
+        this.catchPhrase = instructorAttrs.catchPhrase;
```

Comment:
So here you initially say that the Person class constructor is going to be expecting 3 parameters but your Instructor constructor only takes one

Reply:
this just means that the Person constructor is going to think that the name that you'll send to the Instructor constructor will be name since that's it's first argument